index.html

```diff
@@ -17,11 +17,15 @@
       <header class="mobile-nav">
         <a href="/" class="logo">Welcome</a>
         <nav>
-          <i style="font-size: x-large" class="fa-solid fa-bars"></i>
-          <ul>
-            <li><a href="#portfolio">Portfolio</a></li>
-            <li><a href="#about">About</a></li>
-            <li><a href="#contact">Contact</a></li>
+          <i
+            style="font-size: x-large"
+            class="fa-solid fa-bars"
+            id="hamburger"
+          ></i>
```

Comment:
- [ ] Great job so far. As part of the requirements, kindly display the close `X` icon to close the mobile menu

Reply:
Thank you @m05e5